### PR TITLE
Patch broken collection-states merge

### PR DESCRIPTION
**Pull request by eldrickb**
When I was fixing merge conflicts I got sloppy and didn't run the app before I merged, and accidentally broke the build. This PR fixes it. Also includes the one-line implementation of RTL support in the prompts component.

### CODE REVIEW

File: ui/src/StateContainer.js [hunk 1]
```diff
@@ -251,22 +253,21 @@ export class StateContainer extends LitElement {
...
-            @end-session="${this.handleEndSession}"
             @first-access-over="${this.handleFirstAccessOver}"
+            @skip-prompt="${this.handleSkipPrompt}"
             @update-wave="${this.handleUpdateWave}"
+            @update-user-info="${this.handleUserInfoUpdate}"
+            @update-collection-state=${this.updateCollectionState}
         >
             ${this.renderToast()}
             <vox-view-container
-                .audioStream=${this.audioStream}
```
Comment (ashleygranitto) [inline ui/src/StateContainer.js:263]: nit: alphabetize

File: ui/src/StateContainer.js [hunk 2]
```diff
@@ -251,22 +253,21 @@ export class StateContainer extends LitElement {
             @end-session="${this.handleEndSession}"
             @enter-form="${this.handleEnterForm}"
             @exit-form="${this.handleExitForm}"
-            @update-user-info="${this.handleUserInfoUpdate}"
-            @update-collection-state=${this.updateCollectionState}
-            @skip-prompt="${this.handleSkipPrompt}"
-            @end-session="${this.handleEndSession}"
```
Comment (ashleygranitto) [inline ui/src/StateContainer.js:257]: just double checking--was removing @end-session intentional?

Comment (eldrickb) [reply ui/src/StateContainer.js:257]: it's still there

File: ui/src/components/Prompts.js [hunk 3]
```diff
@@ -67,7 +67,7 @@ export class Prompts extends LitElement {
     renderPromptType() {
         switch (this.prompt.type) {
             case 'TEXT':
-                return html`<p>${this.prompt.body}</p>`;
+                return html`<p dir="auto">${this.prompt.body}</p>`;
```
Comment (mmallett) [inline ui/src/components/Prompts.js:70]: очень хорошо!

File: ui/src/components/RecordingSection.js [hunk 4]
```diff
@@ -26,28 +26,28 @@ export class RecordingSection extends LitElement {
...
 
-    updated(changedProperties) {
-        if (
-            changedProperties.get('collectionState') ===
-            CollectionStates.RECORDING
-        ) {
-            this.isRecording = true;
-        }
+    constructor() {
+        super()
```
Comment (mmallett) [inline ui/src/components/RecordingSection.js:37]: nit semicolon

**Pull request merged**